package.json

```diff
@@ -181,6 +182,7 @@
 		"mockery": "2.1.0",
 		"nock": "10.0.6",
 		"node-wp-i18n": "1.2.3",
+		"postcss-custom-properties": "8.0.10",
```

Comment:
Do we need this dependency directly, or should we rely on `calypso-build` to pull it in?

Reply:
If we directly call on it, keep it. If it is only through calypso-build's provided functionality, I would say remove it.

Otherwise, we'd get in situations where we'd update postcss-custom-properties out of sync with calypso-build, just adding more bulk to the build process without reason.